cloudlaunchserver/values.yaml

```diff
@@ -89,7 +89,8 @@ affinity: {}
 
 rabbitmq:
   rabbitmqUsername: notAGuest
```

Comment:
We could make the username cloudlaunch perhaps?

Reply:
Sure!

Reply:
Changed while merging commit.